extra/yassl/include/openssl/crypto.h

```diff
@@ -19,7 +19,7 @@
 
 /* crypto.h for openSSL */
 
-#ifndef ysSSL_crypto_h__
+#ifndef yaSSL_crypto_h__
```

Comment:
Please report upstream if present in 5.6 at least

Reply:
It isn't. The yassl versions is different in the 3 versions, and I've only seen (different) issues in 5.5 and 5.7